Cargo.lock

```diff
@@ -119,7 +119,7 @@ name = "bigint"
 version = "3.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "byteorder 1.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "byteorder 1.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Sure, i was talking about cases where dependencies upgraded for no actual reason, which is obviously not a case here!